pygtracker/pygtracker.py

```diff
@@ -23,6 +24,23 @@ def __init__(self):
     # End: Summarize the grades by courses
 
     # Start: Summarise the grades by students
+    def student_ranker(course_id="all", n=10, descending=True):
+        """[Calculates the average grade for a specified number of students and ranks them in descending order for a specific course or for the entire program completed thus far]
+
+        Args:
+            course_id (int): [the course ID for which the ranking is calculated for by student.
+            The default option will calculate the ranking for the entire program]. Defaults to "all".
```

Comment:
Some ways to declare [optional values](https://numpydoc.readthedocs.io/en/latest/format.html) for parameters:

```python
copy : bool, default True
copy : bool, default=True
copy : bool, default: True
```